### PR TITLE
Bug/tdi 38437 marketo schema propagate

### PR DESCRIPTION
**Pull request by undx**
**Please check if the PR fulfills these requirements**

- [X] The commit message follows Talend standard
- [x] Tests for the changes have been added (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

Bug in M2, the schema on TMarketoInput doesn't propagate on next components.

**What is the new behavior?**

Schema should propagate now.

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

The TMarketoInput's getMultipleLeads operation have to be rewritten (a virtual component may needed).

### CODE REVIEW

**Comment by sgandon** [comment]
@undx , could you fill the desctiption template for the PR please to explain what you did.

File: components/components-marketo/src/main/java/org/talend/components/marketo/MarketoConstants.java [hunk 1]
```diff
@@ -246,7 +246,7 @@ public static Schema getRESTSchemaForGetLeadActivity() {
     /////////////////////////////////////////////////////////////////////////
     public static Schema getSOAPSchemaForGetLeadOrGetMultipleLeads() {
         return SchemaBuilder.builder().record("getLeadOrGetMultipleLeads").fields() //
-                .name("Id").prop(SchemaConstants.TALEND_COLUMN_IS_KEY, "true").type().nullable().longType().noDefault() //
+                .name("Id").prop(SchemaConstants.TALEND_COLUMN_IS_KEY, "true").type().nullable().intType().noDefault() //
```
Comment (sgandon) [inline components/components-marketo/src/main/java/org/talend/components/marketo/MarketoConstants.java]: I don't see any unit tests on this change ?

Comment (undx) [inline components/components-marketo/src/main/java/org/talend/components/marketo/MarketoConstants.java]: You're right. Tests are coming. This was a really quick fix...

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/18/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/18/)

**Comment by sgandon** [comment]
LGTM, please merge.

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/20/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/20/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/32/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/32/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/33/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/33/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/34/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/34/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/9/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/9/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/12/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/12/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/13/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/13/)

**Comment by sgandon** [comment]
please merge.

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/14/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/14/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/15/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/15/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/19/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/19/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/22/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/22/)

**Pull request merged**